extensions/wikia/PortabilityDashboard/PortabilityDashboard.setup.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+$dir = dirname( __FILE__ ) . '/';
+
+$wgAutoloadClasses[ 'PortabilityDashboardModel' ] = $dir . 'PortabilityDashboardModel.class.php';
+$wgAutoloadClasses[ 'PortabilityDashboardHooks' ] = $dir . 'PortabilityDashboardHooks.class.php';
+$wgAutoloadClasses[ 'SpecialPortabilityDashboardController' ] = $dir . 'SpecialPortabilityDashboardController.php';
+
+$wgSpecialPages[ 'PortabilityDashboard' ] = 'SpecialPortabilityDashboardController';
+$wgSpecialPageGroups[ 'PortabilityDashboard' ] = 'wikia';
+
+$wgHooks[ 'UnconvertedInfoboxesQueryRecached' ][] = 'PortabilityDashboardHooks::updateUnconvertedInfoboxes';
```

Comment:
It's not, should I create a new (always loaded) setup file for those, or add those to some global config file?